homeassistant/components/system_bridge/sensor.py

```diff
@@ -65,44 +65,41 @@ def cpu_speed(data: SystemBridgeCoordinatorData) -> float | None:
 
 def gpu_core_clock_speed(data: SystemBridgeCoordinatorData, key: str) -> float | None:
     """Return the GPU core clock speed."""
-    if getattr(data.gpu, f"{key}_core_clock") is not None:
-        return round(getattr(data.gpu, f"{key}_core_clock"))
+    if value := getattr(data.gpu, f"{key}_core_clock", None) is not None:
```

Comment:
Missing required parenthesizes in all occurrences in this PR:

```suggestion
    if (value := getattr(data.gpu, f"{key}_core_clock", None)) is not None:
```